de_web_plugin.cpp

```diff
@@ -6066,6 +6082,7 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
 
         if (!isDeviceSupported(node, modelId))
         {
+            DBG_Printf(DBG_INFO, "Unsupported device : %s\n", qPrintable(modelId));
```

Comment:
I'd rather not have this in this PR, since this likely causes some confusing debug prints for everything which isn't a sensor and included with help of the underlying list.

Same in line 15079.

Reply:
Done